profile.d/toolbox.sh

```diff
@@ -1,5 +1,9 @@
 # shellcheck shell=sh
 
+# Toolbox provides shim binaries for executing commands on the host from inside
+# of a toolbox container
+export PATH="/usr/libexec/toolbox:$PATH"
```

Comment:
Should we really do this unconditionally for both hosts and containers?  I think this should ideally happen only for Toolbx containers, not even generic Podman containers.  Or did I misunderstand something?